modules/stochastic_tools/test/tests/distributions/gold/normal_out.csv

```diff
@@ -1,2 +1,2 @@
 time,cdf,pdf,quantile
-0,0.5,0.398942280401,0
+0,0.5,0.39894228040143,1.4900501721371e-08
```

Comment:
I'm glad to see that these are almost spot-on the same as the boost version!  I think we should make all of these boost and non-boost distributions diff against the same gold file.